sources/Makefile.am

```diff
@@ -13,9 +13,9 @@ sources =
 
 build_sources = AssemblyInfo.cs $(sources)
 
-CLEANFILES = $(DLL) generated-stamp generated/*.cs $(API)
+CLEANFILES = $(DLL) generated-stamp generated/* $(API)
 
-DISTCLEANFILES = AssemblyInfo.cs $(DLLMAP)
+DISTCLEANFILES = AssemblyInfo.cs
```

Comment:
Okay, I see the reason. The DLL map isn't being generated, it's a static file. That means, it shouldn't be remove on `make distclean`, but it currently is.